src/js/controllers/oppController.js

```diff
@@ -62,149 +62,111 @@ export const oppScreen = async function(state){
                 <ul class="referral-row" id="row-0">
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
-                </ul>
-                <ul class="referral-row" id="row-1">
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
-                    <li class="referral">
-                        <h3 class='opp-category'>SUBJECT CATEGORY</h3>
-                        <div>
-                            <h2>Job Title</h2>
-                            <h3>Company Name</h3>
-                            <h3 class="opp-alumni">from Alumni Name</h3>
-                        </div>
-                    </li>
+
                 </ul>
             </div>
         </div>   
     `;
 
-    elements.container.insertAdjacentHTML('beforeend',oppSetUp);
-    controlOpp(state);
-    console.log("bill function started" + state.user.name);
-
-    document.querySelector('.referral-box').addEventListener('click', e => {
-        const btn = e.target.closest('.referral');
-        //const oppQuery = `?id=${btn.id}`;
-        if(btn) {
-            // clear right screen
-            clearScreen();
-            // render temp 2
-            oppView.renderDetail(state.opp.opps[btn.id]);
-
-            document.querySelector('.back-top').addEventListener('click', e => {
-                const btn2 = e.target.closest('.back-top');
-
-                if(btn) {
-                    // clear right screen
-                    clearScreen();
-                    // render temp 2
-                    billFunction(state,oppSetUp);
+    const billFunction = (state) => {
```

Comment:
Please name appropriately

Reply:
We have finished the change for the next PR.